docs/src/pages/components/Button/button.example.html

```diff
@@ -93,17 +93,39 @@ <h6 class="preview__label">Transparent button</h6>
   Disabled
 </button>
 
-<h6 class="preview__label">Size variant buttons</h6>
-<button type="button" class="ds-c-button ds-c-button--small">
-  Small
-</button>
-<button type="button" class="ds-c-button">Default</button>
-<button class="ds-c-button ds-c-button--big">
-  Big
+<h6 class="preview__label">Landing Page Primary button</h6>
+<button class="ds-c-button ds-c-button--big ds-c-button--primary">
+  Landing Page
 </button>
 
 <h6 class="preview__label">Inverse buttons</h6>
 <div class="example--inverse">
+  <div class="ds-u-margin-bottom--1">
```

Comment:
Maybe we can add some labels here as well, like "Inverse Primary", "Inverse Default", "Inverse Transparent" and "Inverse Landing Page"